test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -96,7 +96,7 @@ string TestFunctionCall::format(
 
 			if (m_call.omitsArrow)
 			{
-				if (_renderMode == RenderMode::ActualValuesExpectedGas && (m_failure || !matchesExpectation()))
+				if (_renderMode == RenderMode::ActualValues && _gasRenderMode == GasRenderMode::ExpectedGas && (m_failure || !matchesExpectation()))
```

Comment:
I'm not sure if this properly handles the case of `_renderMode == RenderMode::ActualValues && _gasRenderMode == GasRenderMode::ActualGas`. Before your change this case was impossible but now is.

Looking at how it worked in 8c51a089d7b198ddf26a9c181fb133716826c0f6 (i.e. before @mijovic added this enum in 2b14efbbcc4de3bde7bdc7fae3e951d8bd249ed0), I'd be inclined to say that this should ignore the gas enum but please verify that this is the right behavior:

```suggestion
				if (_renderMode == RenderMode::ActualValues && (m_failure || !matchesExpectation()))
```